tests/embedding_lookup.c

```diff
@@ -0,0 +1,29 @@
+#include <finalfusion.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <limits.h>
+#include <sys/param.h>
```

Comment:
Forgot this one, `sys/param.h` is system-dependent, does not exist on Visual C++. Not sure what it is needed for?

Reply:
All of these are gone now, that was copy-pasted from an example to print the current working directory.